parser-typechecker/src/Unison/Runtime/Interface.hs

```diff
@@ -407,50 +407,50 @@ executeMainComb init cc =
 bugMsg :: PrettyPrintEnv -> Text -> Term Symbol -> Pretty ColorText
 bugMsg ppe name tm
   | name == "blank expression" =
-      P.callout icon . P.lines $
```

Comment:
I think Dan used to use an old version of ormolu, so this is just re-indenting based on that. We chatted on slack and I think he upgraded so hopefully this won't happen again 🤞🏼